app/controllers/application_controller.rb

```diff
@@ -16,14 +18,26 @@ class ApplicationController < ActionController::API
            status: :internal_server_error
   end
 
-  rescue_from ActiveRecord::SubclassNotFound do |exception|
+  rescue_from ActiveRecord::SubclassNotFound, UnknownOrder do |exception|
     render_unprocessable_entity exception
   end
 
   def paginate(scope)
     scope.paginate(:per_page => params[:per_page], :page => params[:page])
   end
 
+  # rubocop:disable Metrics/AbcSize
+  def order(scope, default_order, allowed_keys)
+    if params[:order]
+      order, direction = params[:order].split(' ', 2)
+      unless (direction && %w[asc desc].include?(direction.downcase)) && allowed_keys.include?(order)
+        raise UnknownOrder, "Unknown sort order '#{params[:order]}'"
+      end
+    end
+    scope.order(params[:order] || default_order)
```

Comment:
single string values are deprecated in rails 5, the better option would be using a hash: `order(column: :asc)`